packages/terra-modal/src/ModalOverlay.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+import './ModalOverlay.scss';
+
+const propTypes = {
+  classNameOverlay: PropTypes.string,
```

Comment:
Is there a good reason to keep this as an explicit prop rather than just spreading custom props onto the overlay?

I think you're using className above to pass the class, and not actually using this prop.